tests/bootstrap.php

```diff
@@ -0,0 +1,6 @@
+<?php
+if (class_exists('PHPUnit_Framework_TestCase') === false && class_exists('PHPUnit\Framework\TestCase') === true) {
```

Comment:
Small thing, but can you add a blank line just after `<?php`? And add a new line at the end of the file?